tests/integration_tests/db_engine_specs/trino_tests.py

```diff
@@ -14,11 +14,35 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+import json
+from unittest.mock import Mock
+
 from sqlalchemy.engine.url import URL
 
 from superset.db_engine_specs.trino import TrinoEngineSpec
 from tests.integration_tests.db_engine_specs.base_tests import TestDbEngineSpec
 
+TEST_CERT = """\
+-----BEGIN CERTIFICATE-----
```

Comment:
wondering if some security automation system on our build systems at Preset is going to trigger while seeing this as it looks like stuff that shouldn't get checked into a repo

Reply:
Yeah. I already change from using fake `TEST_CERT` to using mock.
You should take a look